src/useSpeech/__tests__/dom.ts

```diff
@@ -0,0 +1,41 @@
+import { renderHook } from '@testing-library/react-hooks/dom';
+import { useSpeech } from '../index';
+
+describe('useSpeech', () => {
+	// Mock the SpeechSynthesisUtterance and speechSynthesis objects
+	beforeAll(() => {
+		window.SpeechSynthesisUtterance = jest.fn();
+		// @ts-expect-error testing irrelevant usage
```

Comment:
not true - you're mocking Web API
better to place this code within `src/__tests__` folder like it is done to vibrate API

Reply:
@xobotyi Addressed [here](https://github.com/react-hookz/web/pull/1378/commits/aa2088320e58a386ec534b6064ba4f912373c0a8)